.docs/dev-guide/build-reference.md

```diff
@@ -4,7 +4,56 @@ How to build libStorage
 
 ---
 
-## Build Requirements
+## Basic Builds
+The following one-line command is the quickest, simplest, and most
+deterministic approach to building libStorage:
+
+```bash
+$ git clone https://github.com/codedellemc/libstorage && make -C libstorage
+```
```

Comment:
Hi @codenrhoden,

I thought about that. I even had a brief explanation under the command with what it does exactly, but I didn't like the way it crunched the formatted code between two paragraphs. It was distracting visually. I may add another section underneath that with a proviso or use the note formatting block to indicate the explanation.